Include/cpython/code.h

```diff
@@ -2,9 +2,16 @@
 #  error "this header file must not be included directly"
 #endif
 
+/* Each instruction in a code object is a fixed-width value,
```

Comment:
Why do we need this comment? Isn't it documented elsewhere?

Reply:
If it is, it isn't where I looked.  At the least it would be helpful for this to be a comment referring to existing documentation, if such exists.

Reply:
I did another check and did not find any documentation of the wordcode format, neither in the repo (incl. docs) nor in the devguide.  Honestly, that would be worth adding, but not in this PR. 🙂  I've opened faster-cpython/ideas#56 to address the matter separately.